core/auth/oauth/oauth.go

```diff
@@ -3,9 +3,9 @@ package oauth
 import (
 	"context"
 	"encoding/gob"
+	"golang.org/x/oauth2"
```

Comment:
Revert this, oauth2 is not part of the standard library

Reply:
What do you suggest do use instead? Do you have any hints for us?

Reply:
Just (re-)order the imports ;)